modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2109,6 +2119,30 @@ public void testDeleteObjectEntry() throws Exception {
 
 		_assertCount(0);
 
+		// Delete entry when definition is related to a draft definition
```

Comment:
Since all other comments in this method have "object entry", keep it here:
`// Delete object entry ...`